components/admin/spec/controllers/matches_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe MatchesController, type: :controller do
+
+  let(:valid_attributes) { { taxonomies: { 1 => [{'tagId'=>1}] } } }
+
+  describe 'POST #create' do
+    it 'creates a new Match' do
+      pending 'Need to seed the test db'
```

Comment:
This should come next - a seeded test db could yield some easy but useful tests.

I wanted to capture at least the parameter format for now.